msmbuilder/cluster/agglomerative.py

```diff
@@ -227,3 +275,4 @@ def fit_predict(self, X):
 
 class LandmarkAgglomerative(MultiSequenceClusterMixin, _LandmarkAgglomerative, BaseEstimator):
```

Comment:
What's the point of having `_LandmarkAgglomerative` as a class rather than just defining everything under the public class, `LandmarkAgglomerative`?